tb/dup_stream_top.cpp

```diff
@@ -2,10 +2,15 @@
 using namespace hls;
 #include "ap_int.h"
 #include "bnn-library.h"
```

Comment:
It looks like you could reduce these includes to `hls_stream.h` only. Please, do if possible.